### PR TITLE
qa: make test_perf_stats_stale_metrics check only the clients created for the tests

### PR DESCRIPTION
**Pull request by joscollin**
Uses the client's global id to get the metrics, instead of using the index.
This ensures that test_perf_stats_stale_metrics checks only the clients mounted for
the tests. Now test_perf_stats_stale_metrics checks only the clients that we spread the IO.

Fixes: https://tracker.ceph.com/issues/54971

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [x] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [x] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins test windows

File: qa/tasks/cephfs/test_mds_metrics.py [hunk 1]
```diff
@@ -405,13 +405,13 @@ def test_perf_stats_stale_metrics(self):
         log.debug("metrics={0}".format(metrics))
         self.assertTrue(valid)
 
+        #mount_a and mount_b are the clients mounted for TestMDSMetrics. So get their
+        #entries from the global_metrics.
+        client_a_name = "client.{0}".format(self.mount_a.get_global_id())
+        client_b_name = "client.{0}".format(self.mount_b.get_global_id())
```
Comment (mchangir) [inline qa/tasks/cephfs/test_mds_metrics.py:411]: if possible, please use `f""` string format for formatted stirngs
its a bit more intuitive to understand

**Comment by joscollin** [comment]
jenkins test windows

**Review by mchangir — COMMENTED** [review]
LGTM

**Comment by joscollin** [comment]
@vshankar ping

**Review by vshankar — APPROVED** [review]

**Comment by joscollin** [comment]
http://pulpito.front.sepia.ceph.com/jcollin-2022-03-30_04:12:05-fs-wip-B54971-rank0-stale-perf-stats-assertion-error-distro-default-smithi/

**Review by vshankar — APPROVED** [review]
https://tracker.ceph.com/projects/cephfs/wiki/Master#2022-Apr-14

**Pull request merged**